package.json

```diff
@@ -96,7 +96,7 @@
     "normalize.css": "^8.0.0",
     "nypr-account-settings": ">=0.2.2",
     "nypr-ads": ">=0.1.0",
-    "nypr-audio-services": ">=0.4.0",
+    "nypr-audio-services": "nypublicradio/nypr-audio-services#a92c9701cff2fc6069e77186664c9f1e2800d892",
```

Comment:
Let's hold on merging this to master until the new `nypr-audio-services` version is published, so you can just use the version number.

Reply:
done!